docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -472,8 +472,44 @@ <h1>Samza Configuration Reference</h1>
                             <dt><code>org.apache.samza.coordinator.AzureCoordinationUtilsFactory</code></dt>
                             <dd>Azure based coordination utils.</dd>
                         These coordination utils are currently used for intermediate stream creation.
+                        </dl>
                     </td>
                 </tr>
+
+                <tr>
+                    <td class="property" id="job.logged.store.path">job.logged.store.path</td>
+                    <td class="default">
+                        <dl>
+                            <dt><i>user.dir</i> environment property</dt>
+                            <dt>If <i>user.dir</i> is not defined, <i>CWD</i> is used</dt>
+                        </dl>
+                    </td>
+                    <td class="description">
+                        The base directory for changelog stores used by Samza application. It allows users to configure the base directory at a job level. Another way to configure the base directory is by setting environment variable <i>LOGGED_STORE_BASE_DIR</i>.
+                        <b>Note:</b> The environment variable takes precedence over <i>job.logged.store.path</i>.
+
+                        <br>By opting in, users are responsible for cleaning up the store directories if necessary. For jobs with host-affinity, make sure the changelog stores are persisted across application restarts to minimize restoration time.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="job.non-logged.store.path">job.non-logged.store.path</td>
+                    <td class="default">
+                        <dl>
+                            <dt><i>user.dir</i> environment property</dt>
+                            <dt>If <i>user.dir</i> is not defined, <i>CWD</i> is used</dt>
+                        </dl>
+                    </td>
+                    <td class="description">
+                        The base directory for non-changelog stores used by Samza application. It allows users to configure the base directory at a job level.
+
+                        <br>For YARN, the current behaviour without the configuration is to create non-changelog store directories in CWD which happens to be the yarn container directory. As part of NodeManager's deletion service, the data gets cleaned up
+                        periodically, which is controlled by the YARN config <i>yarn.nodemanager.delete.debug-delay-sec</i>.
+
+                        <br>By opting in, users are responsible for cleaning up the store directories if necessary.
```

Comment:
Maybe: "When set to something other than the YARN container directory, or running in a different environment, these stores should be cleaned up periodically."